lib/govuk_navigation_helpers/tasklist/ab_testable.rb

```diff
@@ -0,0 +1,52 @@
+module GovukNavigationHelpers
+  module Tasklist
+    module AbTestable
```

Comment:
Should we rename this to `SidebarAbTestable` to match the `HeaderAbTestable` style?